src/math/trigonometry.js

```diff
@@ -79,6 +145,19 @@ define(function (require) {
    * @param  {Number} y y-coordinate of the point
    * @param  {Number} x x-coordinate of the point
    * @return {Number}   the arc tangent of the given point
+   *
+   * @example
+   * <div>
+   * <code>
+   * function draw() {
+   * background(204);
+   * translate(width/2, height/2);
+   * var a = atan2(mouseY-height/2, mouseX-width/2);
+   * rotate(a);
+   * rect(-30, -5, 60, 10);
+   * }
```

Comment:
oops, i didn't see the rest of these comments. i'm going to sit down with @chst1 in a moment and we'll do it together.